src/ocean/task/util/StreamProcessor.d

```diff
@@ -184,6 +184,7 @@ class StreamProcessor ( TaskT : Task )
 
         if (throttler_config.resume_point == size_t.max)
             throttler_config.resume_point = total / 5;
+		else
```

Comment:
My original intention was exactly like @gkotian describes - to always do enforce, even if it is a bit redundant (because it is more refactoring-friendly). There is no bug, just broken formatting leftover from old code structure.